lib/ohai/plugins/c.rb

```diff
@@ -39,6 +40,7 @@
         end
       end
     rescue Errno::ENOENT
```

Comment:
I'm going to clean up the other places we have this issue.  Debug output for Ohai is hard to read right now